cumulusci/robotframework/salesforce_robot_library_base.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from cumulusci.core.config import TaskConfig
+from robot.libraries.BuiltIn import BuiltIn
+from cumulusci.tasks.apex.anon import AnonymousApexTask
+
+from cumulusci.tasks.apex.batch import BatchApexWait
+
+
+class SalesforceRobotLibraryBase(object):
```

Comment:
I recommend renaming this to `BaseLibrary`. Both `Salesforce` and `Robot` are a bit redundant, and the namespace provides the additional context. This will also make documentation a bit cleaner, as we'll be able to say "use `BaseLibrary` when creating a traditional robot library, and `BasePage` when creating page objects."